org.hdfgroup.HDFView.yml

```diff
@@ -51,11 +52,18 @@ modules:
       - -DBUILD_TESTING:BOOL=OFF
       - -DBUILD_SHARED_LIBS:BOOL=ON
       - -DBUILD_STATIC_LIBS:BOOL=OFF
+      - -DJava_JAVA_EXECUTABLE=/usr/lib/sdk/openjdk/bin/java
+      - -DJava_JAR_EXECUTABLE=/usr/lib/sdk/openjdk/bin/jar
+      - -DJava_JAVAC_EXECUTABLE=/usr/lib/sdk/openjdk/bin/javac
+      - -DJava_JAVAH_EXECUTABLE=/usr/lib/sdk/openjdk/bin/javah
+      - -DJava_JAVADOC_EXECUTABLE=/usr/lib/sdk/openjdk/bin/javadoc
+      - -DJAVA_INCLUDE_PATH=/usr/lib/sdk/openjdk/jvm/openjdk-22/include/
+      - -DJAVA_INCLUDE_PATH2=/usr/lib/sdk/openjdk/jvm/openjdk-22/include/linux/
+      - -DJAVA_AWT_INCLUDE_PATH=/usr/lib/sdk/openjdk/jvm/openjdk-22/include/
```

Comment:
Hummmm, did you investigate why these are suddenly needed? Seems like this is something CMake shoud determine at configure time itself...